hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.tsx

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import styles from "./TeamDetail.module.scss";
+import Header from "components/general/Header/Header";
+import Typography from "@material-ui/core/Typography";
+import { mockTeamMultiple } from "testing/mockData";
+import {
+    Table,
+    TableBody,
+    TableContainer,
+    TableHead,
+    TableRow,
+    TableCell,
+    Checkbox,
+    Paper,
+    Container,
+    Box,
+    Grid,
+} from "@material-ui/core";
+
+const TeamDetail = () => {
+    return (
+        <>
+            <Header />
+            <Container>
+                <Grid container direction="column" spacing={6}>
+                    <Grid item>
+                        <Typography variant="h1">
+                            Team {mockTeamMultiple.id} Overview
+                        </Typography>
+                    </Grid>
+                    {/* Team info and actions */}
+                    <Grid container spacing={2}>
+                        <Grid
+                            container
+                            direction="column"
+                            spacing={1}
+                            item
+                            lg={6}
+                            xs={12}
+                        >
+                            <Grid item>
```

Comment:
Same here, I don't think you need a nested grid for this, the only thing that needs a grid are the 2 tables.